com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/BundleHelper.java

```diff
@@ -864,4 +864,8 @@ public static void extractPipelinePlugins(Project project, String pluginsDir) th
         ExtenderUtil.storeResources(new File(pluginsDir), sources);
     }
 
+    public static boolean isResourcesNeeded(Project project) {
```

Comment:
Not sure about the function name. I think it should more closely resemble the name of the option it checks. So perhaps `isArchiveExcluded()` or `shouldIncludeArchiveInBundle()`